RELEASE.md

```diff
@@ -1,13 +1,36 @@
 # Release Process
 
-TODO
+## Overview
 
-(text below is from the template)
+The Service APIs project is an API project that has the following two components:
+- Kubernetes Custom Resource Definitions (CRDs)
+- Corresponding Go API in the form of `sigs.k8s.io/service-apis` Go package
 
-The Kubernetes Template Project is released on an as-needed basis. The process is as follows:
+This repository is the home for both of the above components.
 
-1. An issue is proposing a new release with a changelog since the last release
-1. All [OWNERS](OWNERS) must LGTM this release
-1. An OWNER runs `git tag -s $VERSION` and inserts the changelog and pushes the tag with `git push $VERSION`
-1. The release issue is closed
-1. An announcement email is sent to `kubernetes-dev@googlegroups.com` with the subject `[ANNOUNCE] kubernetes-template-project $VERSION is released`
+## Versioning strategy
+
+Releases in this repository follow Go module versioning conventions and
+semantic versioning.
+During the `alpha` and `beta` stage of the API, version tags will take the form
+of `v0.x.y`.
+Minor version must be incremented whenever a new API version is introduced for
+any resource or whenever there are substantial changes in the API.
```

Comment:
Wouldn't we release a new API version any time we made substantial changes to the API? Maybe we can leave that second part out since some might read it to suggest that we will make substantial changes to the API without bumping the API version.

Reply:
Reworded this a little bit.
I think we will certainly add in backwards-compatible additions to the API (even after GA) and users should expect that.

Reply:
👍 that seems reasonable.